confluent_kafka/deserializing_consumer.py

```diff
@@ -43,59 +43,57 @@ class DeserializingConsumer(_ConsumerImpl):
-    |                    |                 | The function argument is a JSON formatted str       |
-    |                    |                 | containing statistics data.                         |
-    +--------------------+-----------------+-----------------------------------------------------+
-    |                    |                 | Callable(ThrottleEvent)                             |
-    | throttle_cb        | callable        |                                                     |
-    |                    |                 | Callback for throttled request reporting.           |
-    +--------------------+-----------------+-----------------------------------------------------+
-
-    .. _See Client CONFIGURATION.md for a complete list of configuration properties:
-        https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+    Notable DeserializingConsumer configuration properties(* indicates required field)
+
+    +----------------------+-----------------+-----------------------------------------------------+
+    | Property Name        | Type            | Description                                         |
+    +======================+=================+=====================================================+
+    | `bootstrap.servers*` | str             | Comma-separated list of brokers.                    |
+    +----------------------+-----------------+-----------------------------------------------------+
+    |                      |                 | Client group id string.                             |
+    | `group.id*`          | str             | All clients sharing the same group.id belong to the |
+    |                      |                 | same group.                                         |
+    +----------------------+-----------------+-----------------------------------------------------+
+    |                      |                 | Callable(SerializationContext, bytes) -> obj        |
+    | `key.deserializer`   | callable        |                                                     |
+    |                      |                 | Deserializer used for message keys.                 |
+    +----------------------+-----------------+-----------------------------------------------------+
+    |                      |                 | Callable(SerializationContext, bytes) -> obj        |
+    | `value.deserializer` | callable        |                                                     |
+    |                      |                 | Deserializer used for message values.               |
+    +----------------------+-----------------+-----------------------------------------------------+
+    |                      |                 | Callable(KafkaError)                                |
+    |                      |                 |                                                     |
+    | error_cb             | callable        | Callback for generic/global error events. These     |
```

Comment:
These should be ticked as well.

The tick-no-tick rule is differentiate between normal descriptive text and programmatic literals that have meaning to the program and must be written in their exact form, such as configuration parameters, signatures, or anyhting really that relates to what ends up in the user code.